build-scripts/cpe-generate.py

```diff
@@ -59,18 +60,25 @@ def collect_nodes(tree, reflist):
     return elementlist
 
 
+def parse_args():
+    p = argparse.ArgumentParser(description="This script takes as input an "
+        "OVAL file and a CPE dictionary file and extracts any inventory "
+        "definitions and the tests, states objects and variables it "
+        "references, and then write them into a standalone OVAL CPE file, "
+        "along with a synchronized CPE dictionary file.")
+    p.add_argument("product", help="Name of the product")
+    p.add_argument("idname", help="Identifier prefix")
+    p.add_argument("cpeoutdir", help="Artifact output directory")
+    p.add_argument("ovalfile", help="OVAL file to process")
+    p.add_argument("cpedictfile", help="CPE dictionary file to process")
+
+    args = p.parse_args()
+
+    return args.product, args.idname, args.cpeoutdir, args.ovalfile, args.cpedictfile
```

Comment:
how about returning args here and then using the args dict in main()? I think that would make it more readable and you'd avoid having to list all the args twice.